_templates/component/new/stories.ejs.t

```diff
@@ -9,4 +9,4 @@ import { <%= connected ? `${nameBase} as ${name}` : name %> } from "."
 
 storiesOf("<%= name %>", module)
   .addDecorator(decoratorCentered)
-  .add("default", () => <<%= name %>><%= name %></<%= name %>>)
+  .add("default", () => <<%= name %> name="test" />)
```

Comment:
```suggestion
  .add("default", () => <<%= name %> name={<%= name %>} />)
```

Reply:
@roblafeve Does the name still need to be wrapped in quotes or does hygen handle that?

Reply:
🤔 oh, maybe? So maybe: 
```suggestion
  .add("default", () => <<%= name %> name="<%= name %>" />)
```